cmd/snaplock/runinhibit/inhibit.go

```diff
@@ -59,23 +61,72 @@ const (
 	HintInhibitedForPreDownload Hint = "pre-download"
 )
 
+const hintFilePostfix = "lock"
+
 // HintFile returns the full path of the run inhibition lock file for the given snap.
 func HintFile(snapName string) string {
-	return filepath.Join(InhibitDir, snapName+".lock")
+	return filepath.Join(InhibitDir, fmt.Sprintf("%s.%s", snapName, hintFilePostfix))
+}
+
+func InhibitInfoFile(snapName string, hint Hint) string {
+	return filepath.Join(InhibitDir, fmt.Sprintf("%s.%s", snapName, hint))
 }
 
 func openHintFileLock(snapName string) (*osutil.FileLock, error) {
 	return osutil.NewFileLockWithMode(HintFile(snapName), 0644)
 }
 
+type InhibitInfo struct {
+	Revision snap.Revision `json:"revision"`
```

Comment:
calling this just Revision makes it not very clear whether revision was the previous current revision or the one we will install?  either the comment for the struct should make this clear, or we should s/Revision/Previous/ ?

Reply:
thank you! I went with the doc comment approach, does it need rewording?

Reply:
My gut-feeling is that maybe `LastRevision` or `Previous` would be slightly nicer.

Reply:
updated to `Previous`